src/utils/validator.util.ts

```diff
@@ -0,0 +1,20 @@
+const isEmpty = (value: null | undefined | string | object): boolean =>
+    value === undefined ||
+    value === null ||
+    (typeof value === 'object' && Object.keys(value).length === 0) ||
+    (typeof value === 'string' && value.trim().length === 0);
```

Comment:
case for when value is array ?

Reply:
- Type of array is object
- Object.keys([]).length will be 0
- so line 4 will take care of both arrays and objects

Reply:
got it